llm_bioinformatics_research/.gitignore

```diff
@@ -12,6 +12,7 @@
 /build
 
 # misc
+.env
```

Comment:
I remember you mentioning that we will have to save our MongoDB credentials within a .env file and that it's in the .gitignore so that it never gets pushed. How would you like us to format this .env file?